apps/templates/content_templates.py

```diff
@@ -335,7 +349,8 @@ def get_item_from_template(template):
     """
     item = template.get('data', {})
     item[ITEM_STATE] = CONTENT_STATE.SUBMITTED
-    item['task'] = {'desk': template.get('template_desk'), 'stage': template.get('template_stage')}
+    if template.get('schedule_desk'):
```

Comment:
Correct user won;t be notified. 

My concern is that the users won't be able to search an item without user and desk. Can we exit the process from creating content and log the problem? Also change the validation for template so that `desk` information is mandatory for scheduled templates.